### ISSUE TITLE
Possible version numbering issue?

### ISSUE THREAD

**Issue by PhysicsDan** [Warwick-Plasma/epoch#462]
Hi I downloaded the most recent EPOCH version 4.19 (using both `git clone` and from the Releases section) however when I run the code it displays:
```
       d########P  d########b        .######b          d#######  d##P      d##P
       d########P  d###########    d###########     .##########  d##P      d##P 
      ----        ----     ----  -----     ----   -----         ----      -- P  
     d########P  d####,,,####P ####.      .#### d###P          d############P   
    d########P  d#########P   ####       .###P ####.          d############P    
   d##P        d##P           ####     d####   ####.         d##P      d##P     
  d########P  d##P            ###########P     ##########P  d##P      d##P      
 d########P  d##P              d######P          #######P  d##P      d##P       

 Welcome to EPOCH1D version 4.18.0   (commit v4.18.0-51-gf3712586-clean)

 *************************************************************
 The code was compiled with no compile time options
 *************************************************************
 Code is running on 1 processing elements
```
as well as the release .zip wile when unzipped shows epoch-4.18

I checked the CHANGELOG.md which shows,
```
## v4.18.0 to v4.19.0

 * Ionisation routines have been upgraded, and now read binding energies from...
```
so I believe I actually have version 4.19

**Comment by Status-Mirror** [comment]
Hey Dan,

There is definitely a numbering issue. Unfortunately that welcome screen line isn't written by the source-code, and is set by variables buried in the Makefile and tied to Git. So far I haven't been able to figure out what it's reading or how to change it.

There have been some small bug fixes since the EPOCH 4.19 release, so I would advise you to stick with the `git clone` version.

In the meantime, I'll continue backtracing this error whenever I'm between tasks.

Cheers,
Stuart

**Comment by Status-Mirror** [comment]
Thanks to assistance from Tom Goffrey, I've solved the problem. If performing a new git clone, the code will now correctly display the v4.19.1 (which contains a few bug fixes since the release of v4.19.0). EPOCH did not recognise my previous unsigned version tag.

In future, we can add a tag to the main branch after a commit by performing a *signed* tag locally using

`git tag -s 'v4.19.XX'`

And pushing it to the main branch using 

`git push --tags`

Signed tags must be performed using the GPG service.
